test/controllers/CurrencyOperationsTest.java

```diff
@@ -48,8 +48,8 @@ public void getsCurrencyFromConfiguration() {
     public void throwsExceptionWhenInvalidCurrency() {
         try {
             opsWithCurrency("INVALID").currency();
-            fail("DefaultCurrencyNotFound exception expected with invalid currency");
-        } catch (DefaultCurrencyNotFound e) {
+            fail("DefaultCurrencyNotFoundException exception expected with invalid currency");
```

Comment:
Could you rewrite this?
With assertj 3.x we have much better possibilities to express exceptional behaviour:

```java
assertThrownBy(() -> opsWithCurrency("INVALID").currency())
    .isInstanceOf(DefaultCurrencyNotFoundException.class);
```